scripts/issue-genesis-utxo-expenditure.sh

```diff
@@ -32,6 +32,10 @@ args=" --real-pbft
        --wallet-key          ${from_key}
        --rich-addr-from    \"${from_addr}\"
        --txout            (\"${addr}\",${lovelace})
+       --topology            configuration/simple-topology.json
+       --genesis-file       \"${genesis_file}\"
+       --database-path       ./db/
+       --socket-dir          ./socket/
```

Comment:
Likewise, why are these arguments necessary now?

I think this is quite wrong.

Reply:
Agreed. The cli commands that now include the `nodeCliParser` all eventually call `withRealPBFT` which called `fromProtocol`. `fromProtocol` requires a genesis hash, genesis filepath and pbft sig threshold. I could have modified these functions so they only required what they actually needed (and not previously a `CardanoConfiguration` or currently a `NodeConfiguration`) however I didn't want to increase the size of this PR any further. This is going to by my next PR.

Reply:
I see, thank you Jordan!

Looking forward for the future PRs! : -)